CHANGELOG.md

```diff
@@ -23,6 +26,7 @@ The format is based on [Keep a Changelog].
   any other command that uses the minibuffer. This has now been fixed
   ([#55]).
 
+[#59]: https://github.com/raxod502/ctrlf/issues/59
```

Comment:
This should be at the end of the list for alphabetization.

Reply:
done